### DISCUSSION TITLE
Notices with WinnerSelectionStatus "No winner was chosen and the competition is closed"

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#680] [Q&A] [answered]
Previously, with old TED submissions, when a notice of type F03 and Not Awarded was sent, the Notice reference (NO_DOC_EXT node) of the previous publication (of the previous notice) had to be indicated within the Information on non-award (PROCUREMENT_DISCONTINUED node).

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/5c585cca-b682-4a0f-a35b-c478b442660b)

Now, when we send a Notice with WinnerSelectionStatus "No winner was chosen and the competition is closed" does that information have to be indicated somewhere? We have looked at publications in production of this type but we do not see information of that type.

For example:

[https://ted.europa.eu/udl?uri=TED:NOTICE:574402-2023:TEXT:ES:HTML](url)
[https://ted.europa.eu/udl?uri=TED:NOTICE:544949-2023:TEXT:ES:HTML](url)

Thanks in advance

**Comment by YvesJo** [comment, marked as answer]
Hi,
In eForms, all the  notices (competition, result ...) of a given procedure share the same Procedure ID and there is no need anymore to identify the previous notice.
There are few specific cases where an explicit reference to a previous notice is expected; For further details, you may have a look at the answer to question ["8. What are the cases when a reference to a specific notice is expected?" of the FAQ](https://docs.ted.europa.eu/home/eforms/FAQ/index.html#_forms_and_procedures)
